services/polymart/polymart-downloads.service.js

```diff
@@ -0,0 +1,36 @@
+import { NotFound } from '../../core/base-service/errors.js'
+import { renderDownloadsBadge } from '../downloads.js'
+import { BasePolymartService, documentation } from './polymart-base.js'
+
+export default class PolymartDownloads extends BasePolymartService {
+  static category = 'downloads'
+
+  static route = {
+    base: 'polymart/downloads',
+    pattern: ':resourceId',
+  }
+
+  static examples = [
+    {
+      title: 'Polymart Downloads',
+      namedParams: {
+        resourceId: '323',
+      },
+      staticPreview: renderDownloadsBadge({ downloads: 655 }),
+      documentation,
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'downloads',
+    color: 'green',
+  }
```

Comment:
```suggestion
  static defaultBadgeData = {
    label: 'downloads'
  }
```

I don't think we need to set a default colour here. Either `renderDownloadsBadge()` or throwing a `NotFound()` will set one for us.